onset_functions.py

```diff
@@ -1388,13 +1444,16 @@ def tsa_plot(self, view, selection=None, xlim=None, resample=None):
             sc_identifier = "STEREO-A" if spacecraft[-1] == "a" else "STEREO-B"
 
         if self.spacecraft == "soho":
+            # ERNE-HED (only protons)
             if instrument.lower() == "erne":
                 particle_data = self.current_df_i
                 s_identifier = "protons"
-            if instrument.lower() == "ephin":
```

Comment:
Why have you removed this clear if-clause and replaced it with something ambiguous like "else"? EPHIN has electrons and protons, and we don't know if we want to use EPHIN protons in the future.